cli/cmd/net.go

```diff
@@ -44,15 +43,20 @@ func get(url string, display bool) (map[string]interface{}, error) {
 		panic(err)
 	}
 
-	if err := json.Unmarshal(data, &dat); err != nil {
-		panic(err)
-	}
-
-	if display {
-		s, _ := prettyjson.Marshal(dat)
-		fmt.Println(string(s))
-	}
-	return dat, nil
+  if resp.StatusCode == 200 {
+		var dat map[string]interface{}
+		if err := json.Unmarshal(data, &dat); err != nil {
+			panic(err)
```

Comment:
be carefull with panic, it exits your program

Reply:
Yes, I want to crash when it fails (at least for now)